modules/users/server/controllers/users.profile.server.controller.js

```diff
@@ -80,6 +80,7 @@ exports.userMiniProfileFields = [
   'avatarSource',
   'avatarUploaded',
   'emailHash',
+  'locationLiving',
```

Comment:
There's an API test somewhere for this endpoint, could you add this to there as well?